Documentation/install/system_requirements.rst

```diff
@@ -136,6 +147,26 @@ by adding the following to the helm configuration command line:
      ...
      --set global.enableXTSocketFallback=false
 
+Advanced Features and Required Kernel Version
+=============================================
+Cilium requires Linux Kernel 4.9.17 or higher, however development on additional
+kernel features and functionality continues to progress in the Linux community.
+Some Cilium features and functionality are dependent on newer kernel versions.
+These additional Cilium features and functionality are enabled by upgrading to
+a later kernel version as detailed below:
+
+========================== ===============================
+Cilium Feature             Minimum Kernel Version
+========================== ===============================
+`Longest Prefix Match`_    >= 4.11
+`Host Reachable Services`_ >= 4.19.57, >= 5.1.16+,  >=5.2+
+`Kube-proxy Free`_         >= 4.19.57, >= 5.1.16+,  >=5.2+
+========================== ===============================
+
+.. _Longest Prefix Match: |SCM_WEB|\/install/upgrade/#restrictions-on-unique-prefix-lengths-for-cidr-policy-rules
+.. _Host Reachable Services: |SCM_WEB|\/gettingstarted/host-services/
+.. _Kube-proxy Free: |SCM_WEB|\/gettingstarted/kubeproxy-free
```

Comment:
These three links seem broken. I think they should use the `:ref:` syntax instead, as seen in the e.g. `Key-Value store` section of this same document.

Reply:
fixed, II have been using make render docs but missed these broken links, thanks